appveyor.yml

```diff
@@ -1,7 +1,11 @@
 version: "{build}"
-
 os: Visual Studio 2015
 
+branches:
+  only:
```

Comment:
It is good to see the build status of some branches when developing them. IE while implementing the GRO parser I could see if CI worked for all commits along the way and I could fix issues as I went.  Maybe I'll just do development in my forked and submit PRs from there :smile_cat:

Reply:
> Maybe I'll just do development in my forked and submit PRs from there

Yes, that's what I do. I think you can enable Travis for you fork too